SWMessages/Source/SWMessageView.swift

```diff
@@ -218,6 +231,21 @@ public class SWMessageView :UIView , UIGestureRecognizerDelegate {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public override func drawRect(rect: CGRect) {
+        super.drawRect(rect)
+        let options = SWMessageView.styleForMessageType(notificationType)
```

Comment:
Sorry for late reply, 

styleForMessageType - allows developer to replace global styles for notification types, and styles parameter allows per message style override.

How will this (making styleForMessageType computed property on notificationType) allow global override, and  Custom style per message basis? 

Should we rather consider something like

`    self.appliedStyle = options ?? SWMessageView.styleForMessageType
`

btw, can you give pull request for struct update alone separately? We can decide on API change seperately